matsim/scenariogen/data/__init__.py

```diff
@@ -112,14 +112,16 @@ class HouseholdType(AutoNameLowerStrEnum):
     MULTI_W_CHILDREN = auto()
     MULTI_WO_CHILDREN = auto()
     SINGLE = auto()
+    UNKNOWN = auto()
 
 
 class EconomicStatus(AutoNameLowerStrEnum):
```

Comment:
Do you need to remove the groups ? Why not just add medium_high and medium_low ? Also, please add the underscore to make it more readable.